### PR TITLE
changed sysctl modifications via writing to a file

### LINKED ISSUE #52: duplicated sysctl command
The sysctl commands https://github.com/srl-wim/container-lab/blob/master/clab/docker.go#L99 and  https://github.com/srl-wim/container-lab/blob/master/clab/docker.go#L108 are identical, shouldn't the second one be: 
`b, err = exec.Command("sudo", "sysctl", "-w", "net.ipv4.conf.default.rp_filter=0").CombinedOutput()` ?

### PR DESCRIPTION
**Pull request by hellt**
This PR changes the way we interact with sysctl by swapping subshelling to bash with a write to a respective file

Closes #52 

Additionally, the same approach is used to add LLDP capability to a linux bridge.

### CODE REVIEW

File: clab/docker.go [hunk 1]
```diff
@@ -79,33 +83,23 @@ func (c *cLab) CreateBridge(ctx context.Context) (err error) {
...
-	var b []byte
-	b, err = exec.Command("sudo", "sysctl", "-w", "net.ipv4.conf.all.rp_filter=0").CombinedOutput()
-	if err != nil {
-		return fmt.Errorf("failed to disable Checksum Offloading on docker bridge: %v", err)
-	}
-	log.Debugf("%s", string(b))
-	//if err = sysctl.Set("net.ipv4.conf.default.rp_filter", "0"); err != nil {
-	//	return err
-	//}
-	log.Debug("Disable RPF check on the docker host part2")
```
Comment (hellt) [inline clab/docker.go:92]: @henderiw what was the reason to disable RPF check twice (part1 and part2)? I removed the part2 as it seems unnecessary, unless it is needed for the reasons unknown to me.

Comment (karimra) [reply clab/docker.go:92]: check #52

Comment (hellt) [reply clab/docker.go:92]: Ah, sorry, my bad, I didn't notice it had different scopes in the commands (ipv4 vs default)

Both are needed to disable rpf on existing and new interfaces. I will adjust the code accordingly

Comment (hellt) [reply clab/docker.go:92]: fixed in 7a6585e

**Comment by hellt** [comment]
tested locally

**Pull request merged**